charts/castai-cluster-controller/values.yaml

```diff
@@ -59,13 +59,16 @@ castai:
   clusterID: ""
 
 # podAnnotations -- Annotations added to each pod.
-podAnnotations: {}
+podAnnotations:
+  kubectl.kubernetes.io/default-container: "cluster-controller"
```

Comment:
I would add it directly to the deployment. This is a field for customer override